pyproject.toml

```diff
@@ -24,14 +24,15 @@ numpy = "^1.21.6"
 scipy = "^1.6.1"
 aiohttp = ">=3.10.10"
 multidict = ">=6.0.5"
+qcs-sdk-python = "0.20.1"
```

Comment:
For now this is necessary due to https://github.com/rigetti/qcs-sdk-rust/issues/527

Reply:
I'm not sure I understand why, but doesn't this package have a dependency on `qcs-sdk-python` transitively through `pyquil`?

Reply:
Yes; this is just to lock the dependency at the correct place. In the previous MR, I did the same thing, then removed this line since it's not a direct dependency; but then I had to regenerate the lockfile due to a merge conflict, so I ended up losing the version. So I decided that for now (until we resolve the installation issue) it makes more sense to just have this bound explicit.